helm/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -212,11 +212,7 @@ data:
   tls.crt: {{ $tls.clientCert }}
   tls.key: {{ $tls.clientKey }}
 {{- else }}
-{{- if .Capabilities.APIVersions.Has "cert-manager.io/v1" }}
-apiVersion: cert-manager.io/v1
-{{- else }}
-apiVersion: cert-manager.io/v1alpha2
-{{- end }}
+apiVersion: cert-manager.io/{{ .Values.certManagerApiVersion }}
```

Comment:
Lets specify the v1 version explicitly and we specify the min cert-manager version in our documents.